noxfile.py

```diff
@@ -74,3 +74,19 @@ def dev(session: nox.Session) -> None:
 
     python = os.path.join(venv_dir, "bin/python")
     session.run(python, "-m", "pip", "install", "-e", ".[dev]", external=True)
+
+
+@nox.session(python=["3.9", "3.10", "3.11", "3.12", "pypy3.9", "pypy3.10"])
+def examples(session: nox.Session) -> None:
+    session.install(".[examples]")
+    try:
+        session.install("unstructured[all-docs]")
```

Comment:
[F] This package cannot be installed for all of the operating systems we are testing against. I thought it was better to include it where possible so that we can at least partially cover `examples/llm_and_nlp/unstructured-text.py`.

Reply:
Q: what is `all-docs`? I see that we deal only with pdf in that example, would it simplify / speedup if we use only `[pdf]`?